Add issue policies doc (#17523)
dotnet/aspnetcore: docs/IssueManagementPolicies.md

# Issue Management Policies

We have a lot of issue traffic to manage, so we have a few policies in place to help us do that. This is a brief summary of some of the policies we have in place and the justification for them.

## Commenting on closed issues

In general, we recommend you open a new issue if you have a bug, feature request, or question to discuss. If you find a closed issue that is related, open a *new issue* and link to the closed issue rather than posting on the closed issue. Closed issues don't appear in our triage process, so only the people who have been active on the original thread will be notified of your comment. A new issue will get more attention from the team.

> *In general* we don't mind getting duplicate issues. It's easier for us to close duplicate issues than to discuss multiple root causes on a single issue! We may close your issue as a duplicate if we determine it has the same root cause as another. Don't worry! It's not a waste of our time!

Reviewer: >  Don't worry! It's not a waste of our time!

I put this in because I've have people apologize for "wasting our time" when I mark an issue of duplicate and I'd like to try and make clear that we don't feel it's a waste of our time at all 😄.

## Needs Author Feedback

If a contributor reviews an issue and determines that more information is needed from the author, they will post a comment requesting that information and apply the `Needs: Author Feedback` label. This label indicates that the author needs to post a response in order for us to continue investigating the issue.

> If the author does not post a response within **7 days**, the issue will be automatically closed. If the author responds within **7 days** after the issue is closed, the issue will be automatically re-opened. We recognize that not everyone can respond immediately to our requests, so if you post a response on a closed issue we're happy to take a look.

Reviewer: ```suggestion
If the author does not post a response within **7 days**, the issue will be automatically closed. If the author responds within **7 days** after the issue is closed, the issue will be automatically re-opened. We recognize that you may not be able to respond immediately to our requests, so if you post a response on a closed issue we're happy to take a look.
```
